deltachat-ffi/src/lib.rs

```diff
@@ -2522,7 +2538,7 @@ pub unsafe extern "C" fn dc_msg_set_text(msg: *mut dc_msg_t, text: *mut libc::c_
     }
     let ffi_msg = &mut *msg;
     // TODO: {text} equal to NULL is treated as "", which is strange. Does anyone rely on it?
```

Comment:
FWIW (unrelated to this PR): ffi strings often have equivalency of NULL pointers and empty strings. I wouldn't worry much about it, probably not worth to change it with tons of ffi-using codebases out there.